_includes/navbar.html

```diff
@@ -61,14 +67,20 @@
                 <li><a href="/reference-architecture/" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="reference-architecture">Reference Architecture</a></li>
               </ul>
             </li>
-            <li id="services-nav-item-beta" class="dropdown js-nav-item-central">
+            <li class="dropdown js-nav-item-central">
               <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Services <i class="fa fa-caret-down" aria-hidden="true"></i></a>
               <ul class="dropdown-menu dropdown-menu-left" role="menu">
                 <li><a href="/support/" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="support">Support</a></li>
                 <li><a href="/devops-bootcamp/" ga-on="click" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="devops-bootcamp">DevOps Bootcamp</a></li>
                 <li><a href="/custom-module-development/" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="custom-module-dev">Custom Module Development</a></li>
               </ul>
             </li>
+            <li id="services-nav-item-beta" class="dropdown js-nav-item-central">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Use Cases <i class="fa fa-caret-down" aria-hidden="true"></i></a>
+              <ul class="dropdown-menu dropdown-menu-left" role="menu">
+                <li><a href="/guides/" ga-on="click" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="guides">Development Guides</a></li>
+              </ul>
```

Comment:
Are the items in our nav duplicated?

Reply:
They are currently duplicated because of optimizely experiments where we have two variations of the navbar. This will be reverted once #144 gets merged.

Reply:
Ohhh, understood, thx!